torchrl/data/datasets/minari_data.py

```diff
@@ -243,38 +234,72 @@ def _download_and_preproc(self):
             minari.download_dataset(dataset_id=self.dataset_id)
             parent_dir = Path(tmpdir) / self.dataset_id / "data"
 
-            td_data = TensorDict()
+            td_data = TensorDict({}, batch_size=[])
```

Comment:
those two are equivalent, and I personally like `TensorDict()` more :) 
```suggestion
            td_data = TensorDict()
```